src/humancellatlas/data/metadata/api.py

```diff
@@ -535,7 +545,12 @@ class IpscInductionProtocol(Protocol):
 
 @dataclass(init=False)
 class ImagingProtocol(Protocol):
-    pass
+    target: List[ImagingTarget]
```

Comment:
Didn't we agree on `Set[…]` due to precedent in `Project`?

Reply:
Using a set would collapse all the `ImagingTarget` objects with the same `assay_type` value into 1 object which would work negate our ability to use a `FrequencySetAccumulator` in Azul.

Reply:
List is fine, just briefly explain why in comment above.